pkg/cmd/tknpac/bootstrap/install.go

```diff
@@ -102,6 +103,9 @@ func getDashboardURL(ctx context.Context, opts *bootstrapOpts, run *params.Run)
 	if err := prompt.SurveyAskOne(qs, &ans); err != nil {
 		return err
 	}
+	if _, err := url.ParseRequestURI(ans); err != nil {
```

Comment:
actually with `Parse` even if we don't give `http/https` it will forward and server side it will fail
```
 Do you want me to install the gosmee forwarder? Yes
💡 Your gosmee forward URL has been generated: https://hook.pipelinesascode.com/DEtOpeRldHAD
? Enter your Tekton Dashboard URL:  localhost
? Enter the name of your GitHub application:  
```

but with `ParseRequestURI` it validates